src/gluonts/model/simple_feedforward/_estimator.py

```diff
@@ -204,6 +273,17 @@ def create_training_network(self) -> HybridBlock:
     # we now define how the prediction happens given that we are provided a
     # training network.
     def create_predictor(self, transformation, trained_network):
+        prediction_splitter = InstanceSplitter(
+            target_field=FieldName.TARGET,
+            is_pad_field=FieldName.IS_PAD,
+            start_field=FieldName.START,
+            forecast_start_field=FieldName.FORECAST_START,
+            instance_sampler=TestSplitSampler(),
```

Comment:
Not related to this PR, but the naming is actually confusing, since the `TestSplitSampler` does not actually perform any sampling. Maybe we want to rename this in the future.